pkg/library/flatten/flatten.go

```diff
@@ -127,13 +123,13 @@ func recursiveResolve(workspace *dw.DevWorkspaceTemplateSpec, tooling ResolverTo
 		}
 	}
 
-	resolvedContent, err := overriding.MergeDevWorkspaceTemplateSpec(resolvedContent, resolvedParent, pluginSpecContents...)
+	mergedSpecContent, err := mergeDevWorkspaceElements(resolvedContent, resolvedParent, pluginSpecContents...)
 	if err != nil {
-		return nil, fmt.Errorf("failed to merge DevWorkspace parents/plugins: %w", err)
+		return nil, fmt.Errorf("failed to merge DevWorkspace volumes: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to resolve DevWorkspace: %w", err)
```

Reply:
the details are already in 

```go
	if err := mergeVolumeComponents(main, parent, plugins...); err != nil {
		return nil, fmt.Errorf("failed to merge DevWorkspace volumes: %w", err)
	}
	mergedDevWorkspaceTemplateSpec, err := overriding.MergeDevWorkspaceTemplateSpec(main, parent, plugins...)
	if err != nil {
		return nil, fmt.Errorf("failed to merge DevWorkspace parents/plugins: %w", err)
	}
	return mergedDevWorkspaceTemplateSpec, nil
```

Reply:
Formatting errors continues to be annoying :sweat_smile:

Reply:
yeah, it will never ends.